lib/api/core/plugins/pluginContext.js

```diff
@@ -146,10 +150,9 @@ class PluginContext {
         document: sdk.document,
         index: sdk.index,
         ms: sdk.ms,
-        realtime: new Proxy({}, {
-          get () {
-            throw new PluginImplementationError('Realtime controller is not available in plugins. You should use plugin hooks instead');
-          }
+        realtime: Object.assign(sdk.realtime, {
+          subscribe: throwNotAvailable('realtime.subscribe'),
```

Comment:
Nit: we use `controller:action` notation in the rest of the documentation/kuzzle
```suggestion
          subscribe: throwNotAvailable('realtime:subscribe'),
```